client/verta/verta/operations/monitoring/alert/_alerter.py

```diff
@@ -16,41 +16,36 @@ class _Alerter(object):
 
     _TYPE = _AlertService.AlerterTypeEnum.UNKNOWN
 
-    def __init__(self, comparison):
-        if not isinstance(comparison, comparison_module._VertaComparison):
-            raise TypeError(
-                "`comparison` must be an object from verta.common.comparison,"
-                " not {}".format(type(comparison))
-            )
-
-        self._comparison = comparison
-
     def __repr__(self):
         return "<{} alert>".format(
             _AlertService.AlerterTypeEnum.AlerterType.Name(self._TYPE).lower()
         )
 
-    @property
-    def comparison(self):
-        return self._comparison
-
     @abc.abstractmethod
     def _as_proto(self):
         raise NotImplementedError
 
     @staticmethod
     def _from_proto(msg):
-        comparison = comparison_module._VertaComparison._from_proto(
-            msg.operator,
-            msg.threshold,
-        )
+        _protos_to_classes = {
+            _AlertService.AlertFixed: FixedAlerter,
+            _AlertService.AlertReference: ReferenceAlerter,
+            _AlertService.AlertRange: RangeAlerter,
+        }
```

Comment:
I think there's a way we could have have these proto classes returned by our client classes, so that we could do something like
```python
@classmethod
def _from_proto(cls, msg):
    ...
    _protos_to_classes = {
        subcls._get_proto_class(): subcls
        for subcls
        in cls.__subclasses__
    }
```

Maybe not required here (let me know what you think) but it might reduce later maintenance so we won't need to append to this mapping.